old/main.go

```diff
@@ -228,6 +228,16 @@ func Main() {
 		return
 	}
 
+	// sysfs is required for reading network statistics
+	mntpoint, _ := os.Stat("/sys")
+	parent, _ := os.Stat("/")
+
+	if mntpoint.Sys().(*syscall.Stat_t).Dev == parent.Sys().(*syscall.Stat_t).Dev {
```

Comment:
I think it'd probably be better if this logic went in to a utility function (even better if it could be tested); we normally try to avoid logic in main.go. Maybe linux_backend.Start() would be an appropriate place for something like this?